scripts/pull_readmes.py

```diff
@@ -54,8 +54,8 @@ def parse_markdown_and_replace_links(file_path, baseurl):
                             lambda match: replace_relative_links(match, baseurl),
                             markdown_content, flags=re.DOTALL)
 
-    # Drop angel brackets around hyperlinks because DocuSaurus doesn't like them
-    angel_bracket_pattern = r'<(http.*?)>'
+    # Drop angel brackets around hyperlinks/email links because DocuSaurus doesn't like them
+    angel_bracket_pattern = r'<(http.*?|.*@.*)>'
```

Comment:
not sure why there should be a "*"
```suggestion
    angel_bracket_pattern = r'<(http.+?|.+@.+)>'
```